pycape/cape.py

```diff
@@ -55,16 +54,16 @@ class Cape:
     """A websocket client for interacting with enclaves hosting Cape functions.
 
     This is the main interface for interacting with Cape functions from Python.
-    See module documentation for usage example.
+    See module-level documentation :mod:`pycape.cape` for usage example.
 
     Args:
-        url: The Cape platform websocket URL, which is responsible for forwarding
+        url: The Cape platform's websocket URL, which is responsible for forwarding
             client requests to the proper enclave instances.
-        access_token: Optional string containing a Cape access token generated
-            by the Cape CLI during `cape login`. If None, tries to load the access
-            token from a JSON at "$HOME/.config/cape/auth" (or OS-equivalent path).
-        verbose: Boolean controlling verbose logging for the "pycape" logger.
-            If True, sets log-level to DEBUG.
+        access_token: Optional string containing a Cape access token generated by the
+            Cape CLI during ``cape login``. If None, tries to load the access token from
+            a JSON at ``$HOME/.config/cape/auth`` (or OS-equivalent path).
+        verbose: Boolean controlling verbose logging for the ``"pycape"`` logger.
+            If True, sets log-level to ``DEBUG``.
```

Comment:
You probably have an extra "" around pycape?

Reply:
it's intentional because that's the label you'd pass to `getLogger`